0800-appendix-198116-waterfall.Rmd

```diff
@@ -41,39 +39,32 @@ of Transportation and Infrastructure.
 
 At crossing `r as.character(my_site)`, `r fpr::fpr_my_pscis_info()` is a
 first order stream. Upstream of crossing `r as.character(my_site)`, cutthroat trout and dolly varden have previously been recorded
-[@moe2020KnownBC; @norris2020bcfishobs].
+[@moe2020KnownBC; @norris2020bcfishobs].  
 
 <br>
 
-Waterfall Creek is a primary tributary of the Station/Waterfall Creek
-system. It's located in the community of New Hazelton and is thus
-heavily impacted by urban and industrial development. A restoration
-project was carried out along Waterfall Creek in 2001 by Nortec
-Consulting. To help improve riparian vegetation, 300-400 seedlings were
-planted within 5 m of the creek. General stream enhancement work
-including the construction of walkways and bridges was also conducted
-[@nortecconsulting2000]. In 1990, the Chicago Creek Community
-Environmental Enhancement Society (CCES) developed a rehabilitation
-project that aimed to reintroduce coho salmon to Waterfall Creek using
-transplanted stocks from Toboggan Creek, near Smithers. After the
-transplant license ended, the project had to rely on salmon returning to
-the stream for brood stock to maintain population levels
-[@houlden_donas2002ChicagoCreek]. Irvine (2021) conducted an assessment
-on two crossings located on tributaries to Waterfall Creek.
+Waterfall Creek is a primary tributary to Station Creek. Wilson and Rabnett (2007) reported that the crossing structure located on Highway 16 and Station Creek (PSCIS ID 124420) has been the subject of numerous assessments and designs with respect to the rehabilitation of fish passage. They ranked this crossing as having the highest priority for rehabilitation in the Bulkley River watershed. According to the Gitxsan Watershed Authority, Station Creek's mouth (Xsi Gwin Sagiiblax) was the location of a fishing site called Xsan Xsagiibil [@wilson_rabnett2007FishPassage]. @rabnett_williams2004Highway16 identified this area as having a high priority for additional assessments. To support rehabilitation efforts, @skrconsultantsltd_2006FishPassage carried out a thorough inspection, provided rehabilitation design options, and identified the natural limits of potential fish distribution. 
 
 <br>
 
-Habitat assessments were conducted on a tributary to Waterfall Creek by
-@irvine2022BulkleyRiver in 2021. Located upstream of PSCIS Crossing
-198116, this tributary flows through New Hazelton and crosses the
-highway. The highway crossing and a crossing further downstream were
+Habitat assessments were conducted downstream of the railway on Waterfall and Station Creek by
+@irvine2022BulkleyRiver in 2021. Waterfall Creek flows through New Hazelton and crosses the railway downstream of PSCIS crossing 198117, then flowing under the highway. The highway crossing and a crossing further downstream were
```

Comment:
waterfall flows into Station and _Station_ flows under the highway. Q and basecamp are good tools to facilitate documenting stream names and their relationships to each-other correctly.